### DISCUSSION TITLE
"Next" option (Tab) improvement

### DISCUSSION THREAD

**Discussion by Pelldom** [ekolis/FrEee#223] [Ideas]
Currently, clicking the Next button or the Tab, to cycle through the ships and queues that don't have orders goes through both the ships AND planets. Would be better if they were seperated so you could cycle through ships OR planets without orders/queued items.

Maybe Tab cycles ships and Ctrl+Tab cycles through queues?

**Comment by ekolis** [comment]
Yeah, someone else mentioned that it's annoying to have to cycle through all your queues when you're looking for a ship or fleet, and I actually agree! I was thinking of the Ctrl-Tab thing myself, just never got around to implementing it! I was also thinking of maybe a mini ship/queue list on the side of the screen, since there's all that wasted space around the system map, where idle ships/queues could be displayed on the top of the list. But I'd rather not sink too much effort into the old WinForms GUI, so probably the Ctrl-Tab thing would be as far as it would go for now.

**Reply by Pelldom** [reply]
oooo!! That would be cool!
